sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmark.scala

```diff
@@ -508,6 +548,7 @@ object JsonBenchmark extends SqlBasedBenchmark {
       jsonInDS(50 * 1000 * 1000, numIters)
       jsonInFile(50 * 1000 * 1000, numIters)
       datetimeBenchmark(rowsNum = 10 * 1000 * 1000, numIters)
+      filtersPushdownBenchmark(rowsNum = 100 * 1000, numIters)
```

Comment:
Also, could we have benchmarks for nested cases, too?

Reply:
This PR doesn't support pushdown filters with nested attributes. The benchmarks will be useless. I would prefer to don't add such benchmark in the PR because re-running the benchmark on JDK 8 & 11 takes much time (> 1 hour). And I am not sure what the benchmark will show to us.

Reply:
Any reason not to support the case in this PR?

Reply:
1. I opened this PR on 27 January 2020 before filters w/ nested attrs are supported.
2. End users can get benefits from this PR already
3. Nested filters attr will make this PR more complex, and hard to review.
4. The feature can be implemented separately on top of the changes.

Reply:
Could you leave a TODO comment somewhere to support the case in future?

Reply:
I will do next time when I will address other comments.

Reply:
done